bitnami/grafana/values.yaml

```diff
@@ -304,6 +304,14 @@ grafana:
   ## @param grafana.replicaCount Number of Grafana nodes
   ##
   replicaCount: 1
+  ## @param grafana.kind Use either Deployment or StatefulSet (default)
+  ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
+  ##
+  kind: Deployment
```

Comment:
We should add a validation helper to ensure invalid values are provided, see an example below:

- https://github.com/bitnami/charts/blob/main/bitnami/nats/templates/_helpers.tpl#L93
- https://github.com/bitnami/charts/blob/main/bitnami/nats/templates/_helpers.tpl#L103-L110